modules/oneVideoBidAdapter.js

```diff
@@ -282,7 +289,16 @@ function getRequestData(bid, consentData, bidRequest) {
       bidData.regs.ext.us_privacy = bidRequest.uspConsent
     }
   }
-
+  if (bid.params.video.e2etest) {
+    bidData.imp[0].bidfloor = null;
+    bidData.imp[0].video.w = 300;
+    bidData.imp[0].video.h = 250;
+    bidData.imp[0].video.mimes = ['video/mp4', 'application/javascript'];
+    bidData.imp[0].video.api = [2];
+    bidData.site.page = 'https://verizonmedia.com';
+    bidData.site.ref = 'https://verizonmedia.com';
```

Comment:
Same question as above

Reply:
@jsnellbaker same as above.
Here our ad-server would block domain traffic that is not declaring our platform in the publishers Ads.txt.
By forcing the site and ref we can start testing right away without being dependent on the ad-server Ads.txt crawler logic.